### PR TITLE
fix: duplicate dependency

### PR DESCRIPTION
**Pull request by KurtStam**

### CODE REVIEW

**Comment by KurtStam** [comment]
retest please

File: app/rest/runtime/pom.xml [hunk 1]
```diff
@@ -972,12 +972,6 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
-      <groupId>com.h2database</groupId>
-      <artifactId>h2</artifactId>
-      <scope>test</scope>
```
Comment (zregvart) [inline app/rest/runtime/pom.xml]: Shouldn't we keep this one and remove the `<scope>runtime</scope>` one, we're not depending on h2 at runtime right?

**Comment by zregvart** [comment]
@KurtStam so I don't forget, I can't invite you to CircleCI (I think it has something to do with following/or not the GitHub project), I think the workaround is to open [the link](https://circleci.com/gh/syndesisio/syndesis/tree/pull%2F1733) directly.

File: app/rest/runtime/pom.xml (hunk 1 above)
Comment (KurtStam) [reply app/rest/runtime/pom.xml]: @zregvart y that seems reasonable to me.

**Comment by KurtStam** [comment]
@zregvart RE Circle, using that link didn't help me.

**Pull request merged**